README.md

```diff
@@ -13,31 +13,30 @@ To play, you require *an original AoE II: TC, [AoE II: HD](http://store.steampow
 
 Contact
 -------
-Contact          | Where?
------------------|-------
-Issue Tracker    | [GitHub SFTtech/openage](https://github.com/SFTtech/openage/issues)
-Development Blog | [blog.openage.dev](https://blog.openage.dev)
-Subreddit        | [![reddit](/assets/doc/reddit.svg) /r/openage](https://www.reddit.com/r/openage/)
-Discussions      | [GitHub Discussions](https://github.com/SFTtech/openage/discussions)
-Matrix Chat      | [![matrix](/assets/doc/matrix.svg) `#sfttech:matrix.org`](https://matrix.to/#/#sfttech:matrix.org)
-Money Sink       | [![money sink](/assets/doc/liberapay.svg)](https://liberapay.com/SFTtech)
+| Contact          | Where?                                                                                             |
+| ---------------- | -------------------------------------------------------------------------------------------------- |
+| Issue Tracker    | [GitHub SFTtech/openage](https://github.com/SFTtech/openage/issues)                                |
+| Development Blog | [blog.openage.dev](https://blog.openage.dev)                                                       |
+| Subreddit        | [![reddit](/assets/doc/reddit.svg) /r/openage](https://www.reddit.com/r/openage/)                  |
+| Discussions      | [GitHub Discussions](https://github.com/SFTtech/openage/discussions)                               |
+| Matrix Chat      | [![matrix](/assets/doc/matrix.svg) `#sfttech:matrix.org`](https://matrix.to/#/#sfttech:matrix.org) |
+| Money Sink       | [![money sink](/assets/doc/liberapay.svg)](https://liberapay.com/SFTtech)                          |
 
 
 Technical foundation
 --------------------
 
-Technology     | Component
----------------|----------
-**C++20**      | Engine core
-**Python3**    | Scripting, media conversion, in-game console, code generation
-**Qt6**        | Graphical user interface
-**Cython**     | Python/C++ Glue code
-**CMake**      | Build system
-**OpenGL3.3**  | Rendering, shaders
-**SDL2**       | Cross-platform Audio/Input/Window handling
-**Opus**       | Audio codec
-[**nyan**](https://github.com/SFTtech/nyan) | Content Configuration and Modding
-**Humans**     | Mixing together all of the above
+| Technology                                  | Component                                                     |
+| ------------------------------------------- | ------------------------------------------------------------- |
+| **C++20**                                   | Engine core                                                   |
+| **Python3**                                 | Scripting, media conversion, in-game console, code generation |
+| **Cython**                                  | Python/C++ Glue code                                          |
+| **Qt6**                                     | Graphical user interface                                      |
```

Comment:
maybe just Qt - and we could link to some website for all those techs while we're at it? :)